libraries/Microsoft.Bot.Builder.Dialogs/SkillDialog.cs

```diff
@@ -78,8 +83,10 @@ public override async Task<DialogTurnResult> ContinueDialogAsync(DialogContext d
 
             skillActivity.DeliveryMode = dc.ActiveDialog.State[DeliverModeStateKey] as string;
 
+            var conversationId = (string)dc.ActiveDialog.State[ConversationIdStateKey];
```

Comment:
I realize this might seem trivial but it would probably make this code much clearer if you change the name of this variable to "skillConversationId"

Reply:
done